src/components/Toast/Toast.driver.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { UniDriver } from 'wix-ui-test-utils/unidriver';
+import { TOAST_SKIN } from './Toast';
```

Comment:
The reason the tests are failing, is because we're importing the component's file here.
This causes the protractor tests to import uncompiled files, and they don't know how to deal with it.
We need to move the `TOAST_SKIN` to a different file (maybe the `helpers`), and then use that file here and in `Toast.tsx`